database/migrations/2020_10_05_160539_create_addresses_table.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateAddressesTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('addresses', function (Blueprint $table) {
```

Comment:
All fields except `id`, `contact_name` and the ones created by `->timestampsTz()` are nullable, please add on 
`->nullable()`